server/main.lua

```diff
@@ -440,10 +421,10 @@ RegisterNetEvent('qb-vehicleshop:server:checkFinance', function()
 end)
 
 -- Transfer vehicle to player in passenger seat
-QBCore.Commands.Add('transfervehicle', Lang:t('general.command_transfervehicle'), {{name = 'ID', help = Lang:t('general.command_transfervehicle_help')}, {name = 'amount', help = Lang:t('general.command_transfervehicle_amount')}}, false, function(source, args)
+lib.addCommand('transfervehicle', {help = Lang:t('general.command_transfervehicle'), params = {{name = 'id', type = 'playerId', help = Lang:t('general.command_transfervehicle_help')}, {name = 'amount', type = 'number', help = Lang:t('general.command_transfervehicle_amount')}}}, function(source, args)
     local src = source
-    local buyerId = tonumber(args[1]) --[[@as number]]
-    local sellAmount = tonumber(args[2]) --[[@as number]]
+    local buyerId = args.id
```

Comment:
I think this actually needs to use the tonumber(), have you tested it to see if it works?

Reply:
ox_lib already does this when parsing through the command
![image](https://github.com/Qbox-project/qbx-vehicleshop/assets/22198949/f744376c-6131-4cf0-86ee-f014f3d7c447)

Reply:
quick question as well just to be sure. do we need this event [here](https://github.com/Qbox-project/qbx-vehicleshop/pull/32/files#diff-6f0b2f6b34829d0f65728f2508a4f5e2ef0d39d59b554b3ec8613c8feaacf74dL121)?

Reply:
No